packages/api-core/src/me-settings/mutators.ts

```diff
@@ -32,9 +32,18 @@ export async function updateUserSettings(
 		'two_step_enhanced_security',
 		'primary_site_ID',
 		'mcp_abilities',
+		'user_email_change_pending',
 	];
 	const payload = Object.fromEntries(
 		saveableKeys.filter( ( key ) => key in data ).map( ( key ) => [ key, data[ key ] ] )
 	);
 	return await wpcom.req.post( '/me/settings', payload );
 }
+
+export async function cancelPendingEmailChange(): Promise< Partial< UserSettings > > {
+	return await wpcom.req.post( '/me/settings', { user_email_change_pending: false } );
```

Comment:
nitpick, but I think we could use the `updateUserSettings` function here

Reply:
I called the endpoint intentionally because I don't think we need to go through all the other fields in `userSettings` just to update the 1-2 fields. Plus, it's an actual "action" mutation to cancel the email rather than just updating the settings in general, so I thought it was just best to call the endpoint directly.

Reply:
Yup, fine for me. I don't have a strong opinion about this ;)

Reply:
Ah okay, I misunderstood! I still think the semantics of "canceling" vs "updating" is good to differentiate, but I updated it in [9908200](https://github.com/Automattic/wp-calypso/pull/106103/commits/990820088e056b918dfeba7a9780d05b3b815411)!